### ISSUE TITLE
Site logo notice: "Undefined index: full"

### ISSUE THREAD

**Issue by ardalann** [Automattic/jetpack#2858]
I'm using Jetpack on a WordPress site. 
I just turned on debugging mode and got these notices:
```
Notice: Undefined index: full in /wp-content/plugins/jetpack/modules/theme-tools/site-logo/inc/functions.php on line 66

Notice: Undefined index: full in /wp-content/plugins/jetpack/modules/theme-tools/site-logo/inc/functions.php on line 67
```
This is the ```$_wp_additional_image_sizes``` array:
```
Array
(
    [shop_thumbnail] => Array
        (
            [width] => 180
            [height] => 180
            [crop] => 1
        )

    [shop_catalog] => Array
        (
            [width] => 300
            [height] => 300
            [crop] => 1
        )

    [shop_single] => Array
        (
            [width] => 600
            [height] => 600
            [crop] => 1
        )

)
```
(WooCommerce and StoreFront theme are used in this site)

**Comment by jeherve** [comment]
`$_wp_additional_image_sizes` usually returns at least `thumbnail`, `medium`, `large`, and `full`, but it looks like your theme or one of your plugins.

We should account for that in Jetpack and check if the size exists before to use it.

Thanks for the report!

**Comment by vadikcoma** [comment]
I have the same issue

**Comment by kraftbj** [comment]
@vadikcoma It'll be fixed in the next version of Jetpack, 3.8.1, due out in the next week or so.
